From 23c144b1ed37967871083a23d120922c731b6438 Mon Sep 17 00:00:00 2001
From: Julie Pichon <jpichon@redhat.com>
Date: Mon, 23 Jun 2014 11:58:09 +0100
Subject: [PATCH] Disable broken unit test (related to "Change Password")

The new django_openstack_auth 1.1.6 library surfaced a defect in this
test, whereby the test client does not properly log out the user. This
only affects the test, the functionality otherwise works fine.

This temporarily disables the test in order to repair the gate, while
waiting to replace it with a new unit test more reduced in scope and/or
an integration test.

Change-Id: I8779666c053a4835e22038a39a530fe5fc84b6a3
Partial-Bug: #1333144
(cherry picked from commit 5455b8f6f595633513b6521449ebc1985d5b9070)
---
 openstack_dashboard/dashboards/settings/password/tests.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/openstack_dashboard/dashboards/settings/password/tests.py b/openstack_dashboard/dashboards/settings/password/tests.py
index 6ed948d..d022d6d 100644
--- a/openstack_dashboard/dashboards/settings/password/tests.py
+++ b/openstack_dashboard/dashboards/settings/password/tests.py
@@ -17,6 +17,7 @@
 from django.core.urlresolvers import NoReverseMatch  # noqa
 from django.core.urlresolvers import reverse
 from django import http
+from django.utils import unittest
 
 from mox import IsA  # noqa
 
@@ -53,6 +54,8 @@ class ChangePasswordTests(test.TestCase):
 
         self.assertFormError(res, "form", None, ['Passwords do not match.'])
 
+    # TODO(jpichon): Temporarily disabled, see bug #1333144
+    @unittest.skip("Temporarily disabled, see bug #1333144")
     @test.create_stubs({api.keystone: ('user_update_own_password', )})
     def test_change_password_shows_message_on_login_page(self):
         api.keystone.user_update_own_password(IsA(http.HttpRequest),
